scene/gui/control.cpp

```diff
@@ -1042,55 +1042,37 @@ int Control::get_constant(const StringName &p_name, const StringName &p_type) co
 bool Control::has_icon_override(const StringName &p_name) const {
 
 	const Ref<Texture> *tex = data.icon_override.getptr(p_name);
-	if (tex)
-		return true;
-	else
-		return false;
+	return tex;
```

Comment:
I'd do `tex != NULL` to readability here, implicitly converted a pointer to bool on return is a bit weird IMO.